web/components/MiniRoadmap.test.tsx

```diff
@@ -38,3 +46,48 @@ describe("<MiniRoadmap />", () => {
     expect(subject.queryByText("task1")).not.toBeInTheDocument();
   });
 });
+
+describe("<MiniRoadmap /> with sections", () => {
```

Comment:
i'd prefer to keep with the pattern of a single top-level describe per test, since that's sort of the expected assumption everywhere else

think this could be simpler, you can just re-run useMockRoadmap in your one test